### PR TITLE
Improve SAP Cloud SDK CI/CD tutorials and adapt them to use the General Purpose Pipeline of project "Piper"

### PR DESCRIPTION
**Pull request by daniel-kurzynski**

### CODE REVIEW

File: tutorials/cloudsdk-e2e-test-secure-applications/cloudsdk-e2e-test-secure-applications.md [hunk 1]
```diff
@@ -10,9 +10,9 @@ primary_tag: products>sap-s-4hana-cloud-sdk
 ---
 
 ## Prerequisites
-  - [Set up CI/CD](https://developers.sap.com/tutorials/cloudsdk-ci-cd.html)
-  - [End to end test](https://developers.sap.com/tutorials/cloudsdk-e2e-testing.html)
-  - [Secure your application](https://developers.sap.com/tutorials/s4sdk-secure-cloudfoundry.html)
+  - [Set up CI/CD](cloudsdk-ci-cd)
```
Comment (daniel-kurzynski) [inline tutorials/cloudsdk-e2e-test-secure-applications/cloudsdk-e2e-test-secure-applications.md:13]: @thecodester These changes are already merged to master. Not sure how they made it into this PR.

Comment (daniel-kurzynski) [reply tutorials/cloudsdk-e2e-test-secure-applications/cloudsdk-e2e-test-secure-applications.md]: Fixed by cleaning up the git history.

**Review by thecodester — CHANGES_REQUESTED** [review]
Looks good. 

- The description is too long. If you want to say more, just put it in the intro -- intro starts by skipping line after Your will learn items and then writing whatever you want. Too long description gets cut off on the tutorial tile.

- In Step 3 you have 2 images side by side -- about blue ocean UI. Is that how you want them to appear?

**Comment by daniel-kurzynski** [comment]
> Looks good.
> 
> * The description is too long. If you want to say more, just put it in the intro -- intro starts by skipping line after Your will learn items and then writing whatever you want. Too long description gets cut off on the tutorial tile.
> * In Step 3 you have 2 images side by side -- about blue ocean UI. Is that how you want them to appear?

Thanks for the feedback. Karsten told me that the description should be a bit longer and more speaking. But I now have a little shorter one. 
Please review again.

**Comment by thecodester** [comment]
The description is a 1-sentence, 20-25 word shorthand of what the user will do in the tutorial. It is for the tile, so the tile can only show so much and you want to avoid forcing the user to click in. If you need more information, you can write as much as you need in the intro.

Start the description with the word "Use ..." -- it  reduce the number of words by 25% and get right to the point. After that I will merge.

I am checking the QA version.

**Comment by thecodester** [comment]
Also notice that the code sample (YAML) first one in Step 7 displays very large in the preview -- not sure how you want to fix, but if you put a blank line after the start backticks it fixes this. Don;t think this is why it did not publish but I will take a look on Sunday.

**Comment by daniel-kurzynski** [comment]
> The description is a 1-sentence, 20-25 word shorthand of what the user will do in the tutorial. It is for the tile, so the tile can only show so much and you want to avoid forcing the user to click in. If you need more information, you can write as much as you need in the intro.
> 
> Start the description with the word "Use ..." -- it reduce the number of words by 25% and get right to the point. After that I will merge.
> 
> I am checking the QA version.

Done

**Pull request merged**